src/mlpack/methods/ann/layer/layer_types.hpp

```diff
@@ -200,6 +206,9 @@ class AdaptiveMeanPooling;
 using MoreTypes = boost::variant<
         Glimpse<arma::mat, arma::mat>*,
         Highway<arma::mat, arma::mat>*,
+        MultiheadAttention<SelfAttention, arma::mat, arma::mat>*,
+        MultiheadAttention<EncoderDecoderAttention, arma::mat, arma::mat>*,
+        MultiheadAttention<GeneralAttention, arma::mat, arma::mat>*,
         Recurrent<arma::mat, arma::mat>*,
```

Comment:
@lozhnikov: I am getting an error here -
```
error: ‘SelfAttention’ was not declared in this scope
         MultiheadAttention<SelfAttention, arma::mat, arma::mat>*,
```
and similar for `EncoderDecoderAttention` and `GeneralAttention`. I could not find what I've missed.

Reply:
Ohh...this one is fixed. I was making a silly mistake.